config/development.json

```diff
@@ -89,5 +89,6 @@
 	"BOARD_COLLABORATION_URI": "ws://localhost:4450",
 	"ADMIN_API": {
 		"ALLOWED_API_KEYS": "thisisasupersecureapikeythatisabsolutelysave"
-	}
+	},
+	"TRAINING_URL": "https://lernen.dbildungscloud.de"
```

Comment:
not needed because of default value exists, but it's good to have it for more transparency 👍